packages/zipkin-transport-kafka/src/KafkaLogger.js

```diff
@@ -14,9 +15,15 @@ module.exports = class KafkaLogger {
     const producerOpts = Object.assign({}, producerDefaults, options.producerOpts || {});
     this.producerPromise = new Promise((resolve, reject) => {
       this.topic = options.topic || 'zipkin';
-      this.client = new kafka.Client(
-        clientOpts.connectionString, clientOpts.clientId, clientOpts.zkOpts
-      );
+      if (useZookeeper) {
+        this.client = new kafka.Client(
+          clientOpts.connectionString, clientOpts.clientId, clientOpts.zkOpts
+        );
+      } else {
+        this.client = new kafka.KafkaClient(
+          Object.assign({kafkaHost: clientOpts.connectionString}, clientOpts)
```

Comment:
how about making clientOpts.kafkaHost instead of useZookeeper and hijacking connectionString as the broker host?